src/bci_build/package/basecontainers.py

```diff
@@ -199,6 +200,39 @@ def _get_supported_until_fips(os_version: OsVersion) -> datetime.date:
 ]
 
 
+FIPS_MICRO_CONTAINERS = [
+    OsContainer(
+        name="micro-fips",
+        os_version=os_version,
+        support_level=SupportLevel.L3,
+        supported_until=_get_supported_until_fips(os_version),
+        logo_url="https://opensource.suse.com/bci/SLE_BCI_logomark_green.svg",
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        pretty_name=_get_fips_pretty_name(os_version),
```

Comment:
this is a duplication of the bci-base-fips image. please make it distinct so that we can separate the BASE and micro images.